influunt-api/modules/influunt-core/app/models/Anel.java

```diff
@@ -98,6 +98,9 @@
     private Endereco endereco;
 
     @Column
+    private Boolean aceitaModoManual;
```

Comment:
No banco o campo está como `NOT NULL` para que o modelo fique de acordo precisa de um `@NotNull` no campo.